lua/weapons/swep_sonicsd/modules/cl_sound.lua

```diff
@@ -16,6 +16,7 @@ end
 SWEP:AddHook("Initialize", "sound", function(self)
     self.curbeep=0
     self.eyeangles=Angle(0,0,0)
+    self.mutesonic=false
```

Comment:
Wondering if it might be better to do hooks for this and the prevent reload thing too 

like `if self:CallHook("CanReload") == false then return end` in the main reload code or something and similar for muting sonic like `ShouldPlaySounds`

Note that I'm allowing and encouraging `== false` here because it's different to returning `nil` or not returning anything

Reply:
It probably would be better, but I personally don't actually know how to go about doing that, which is why its the way it is right now

Reply:
Well that is how you do it 😁 gimme a shout on Discord if you get stuck